task14.js

```diff
@@ -0,0 +1,14 @@
+// Write a function that reverses an array without modifying the original array.
+
+function reverseArray(arr) {
+  //   ES6;
```

Comment:
The spread operator creates a shallow copy, which is generally fine for arrays of primitives. However, if the array contains nested objects, the references to those objects will still point to the original ones.

function reverseArray(arr) { 
  return [...arr].reverse(); 
}

const data = [{a:2}, {b:3}, {c:5}]

const reversedArr = reverseArray(data); 

reversedArr[1].b = 10

console.log( data)
console.log(reversedArr)

if u run this data, then after reverse and change data for first element, it will apply for original data as well.